cogs/assistance-cmds/mset9crash.3ds.md

```diff
@@ -3,12 +3,14 @@ title: MSET9 crash recovery
 help-desc: Information on recovering from the infinitely loading Extra Data ARM9 crash (Spinning Square)
 ---
 
-Extra Data stuck at infinitely loading while trying to install Boot9strap with MSET9?
+Extra Data stuck at infinitely loading while trying to install Boot9strap with MSET9, 
```

Comment:
why?

Reply:
> why?

TBH I have no idea what happened there, it should be the question mark